Server/JokesOnYou.Web.Api/Dockerfile

```diff
@@ -0,0 +1,22 @@
+#See https://aka.ms/containerfastmode to understand how Visual Studio uses this Dockerfile to build your images for faster debugging.
```

Comment:
We also have a dockerfile in root, is this intended?

Reply:
Ah, this is a duplicate. Will remove.